src/types/formatted/qaSection.ts

```diff
@@ -0,0 +1,12 @@
+import { FormattedParagraph } from '@/data/queries'
+import { PublishedParagraph } from './publishedEntity'
+
+export type QaSection = PublishedParagraph & {
+  header: string
+  intro: string
+  displayAccordion: boolean
+  questions: FormattedParagraph[]
+  id: string
+  type: string
+  headerTagLevel?: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6'
```

Comment:
There's a type `HeadingLevel` for this defined in `src/types/formatted/headingElement.ts`